doc/concepts/configuration.rst

```diff
@@ -0,0 +1,617 @@
+-   `TT_*` :ref:`environment variables <configuration_environment_variable>`.
+
+This means that `TT_*` environment variables have a higher priority than other configuration ways.
+
+
+
+..  _configuration_options_overview:
+
+Configuration options overview
+------------------------------
+
+This section gives an overview of some useful configuration options.
+All the available options are documented in the :ref:`Configuration reference <configuration_reference>`.
+
+.. _configuration_options_connection:
+
+Connection settings
+~~~~~~~~~~~~~~~~~~~
+
+To configure an address used to listen for incoming requests, use the ``iproto.listen`` option.
+Below are a few examples on how to do this:
+
+*   Set a listening port to ``3301``:
+
+    .. code-block:: yaml
+
+        iproto:
+          listen: "3301"
+
+*   Set a listening address to ``127.0.0.1:3301``:
+
+    .. code-block:: yaml
+
+        iproto:
+          listen: "127.0.0.1:3301"
+
+
+*   Configure several listening addresses:
+
+    .. code-block:: yaml
+
+        iproto:
+          listen: "127.0.0.1:3301,127.0.0.1:3303"
+
+*   Enables :ref:`traffic encryption <enterprise-iproto-encryption>` for a connection using corresponding URI parameters:
+
+    .. code-block:: yaml
+
+        iproto:
+          listen: "127.0.0.1:3301?transport=ssl&ssl_key_file=localhost.key&ssl_cert_file=localhost.crt&ssl_ca_file=ca.crt"
+
+    Note that traffic encryption is supported by the `Enterprise Edition <https://www.tarantool.io/compare/>`_ only.
+
+
+*   Use a Unix domain socket:
+
+    .. code-block:: yaml
+
+        iproto:
+          listen: "unix/:./{{ instance_name }}.iproto"
+
+
+.. _configuration_options_access_control:
+
+Access control
+~~~~~~~~~~~~~~
+
+The ``credentials`` section allows you to grant the specified privileges to users.
+In the example below, there are two users:
+
+*   The *replicator* user is used for replication and has a corresponding role.
+*   The *client* user has the ``super`` role and can perform any action on Tarantool instances.
+
+..  literalinclude:: /code_snippets/snippets/replication/instances.enabled/manual_leader/config.yaml
+    :language: yaml
+    :lines: 1-8
+    :dedent:
+
+To learn more, see the :ref:`Access control <authentication>` section.
+
+
+.. _configuration_options_memory:
+
+Memory
+~~~~~~
+
+The ``memtx.memory`` option specifies how much memory Tarantool allocates to actually store data.
+
+.. code-block:: yaml
+
+    memtx:
+      memory: 100000000
+
+When the limit is reached, ``INSERT`` or ``UPDATE`` requests fail with :ref:`ER_MEMORY_ISSUE <admin-troubleshoot-memory-issues>`.
+
+
+.. _configuration_options_directories:
+
+Snapshots and write-ahead logs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``snapshot.dir`` and ``wal.dir`` options can be used to configure directories for storing snapshots and write-ahead logs.
+
+.. code-block:: yaml
+
+    instance001:
+      snapshot:
+        dir: 'var/snapshots'
+      wal:
+        dir: 'var/wals'
```

Comment:
It worth to split the WAL and snapshot directories, when they're on the separate hard drives. So, I guess, in this case they'll specified as absolute paths.

Reply:
Thanks, updated the paths as follows (and mentioned the recommendation as an example use case):
```
      snapshot:
        dir: '/media/drive1/snapshots'
      wal:
        dir: '/media/drive2/wals'
```